src/clj/puppetlabs/services/ca/certificate_authority_service.clj

```diff
@@ -20,7 +19,10 @@
      (log/info "CA Service adding a ring handler")
      (add-ring-handler
        this
-      (compojure/context path [] (core/build-ring-handler settings puppet-version))))
+       (-> (core/web-routes settings)
+           (#(comidi/context path %))
```

Comment:
I had that in `comidi` at one point (a fn called `context->handler`).  I was following compojure's convention for argument order, and for `context`, compojure puts the `path` arg first.  This means that there's no clean way to use the threading macro on it because all other middleware wrapping tends to expect the ring handler fn as the first arg :disappointed: .

I'd be OK with adding that back to comidi if people want it, but would probably rather deal with that as a separate PR.